test/tasks/build.test.js

```diff
@@ -164,10 +161,7 @@ describe('Build task', function() {
 		});
 
 		afterEach(function() {
-			if (fs.existsSync('build/main.css')) {
-				fs.unlink('build/main.css');
-				fs.rmdir('build');
-			}
+			return exec('rm -rf build');
```

Comment:
Oh, right, I changed it to fs-extra which uses rimraf underneath. (`fs.rmdir`). That needs to stay for other uses, so I think it's fine to have denodeify as well.